### PR TITLE
Feature/user profile

### PR DESCRIPTION
**Pull request by MuresanCoralia**
### Summary

Profile Screen (Xib)

### Metadata
- Ticket link: [Profile Screen](https://trello.com/c/6sSszAJJ/12-ui-profile-screen)
- Relevant design:

### Checks
- [ ] Team members are added to review the Pull Request
- [ ] Dead/unused code is deleted?
- [ ] PR does not break existing functionality
- [ ] Implementation matches the description in the story
- [ ] Images/Gifs have been added for all UI related changes

### CODE REVIEW

File: Internship2022TeamPatri/Internship2022TeamPatri/Models/ColorExtension.swift [hunk 1]
```diff
@@ -12,4 +12,6 @@ import UIKit
      static let colorView = UIColor(red: 0.486, green: 0.478, blue: 0.478, alpha: 1)
      static let colorText = UIColor(red: 0.145, green: 0.0862, blue: 0.0196, alpha: 1)
      static let colorBackground = UIColor(red: 0.752, green: 0.937, blue: 0.952, alpha: 1)
+     static let colorLabelProfile = UIColor(red: 0.164, green: 0.097, blue: 0.022, alpha: 1)
+    static let colorTextProfile = UIColor(red: 0.551, green: 0.542, blue: 0.542, alpha: 1)
```
Comment (carmenj) [inline Internship2022TeamPatri/Internship2022TeamPatri/Models/ColorExtension.swift]: wrong indentation

File: Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/ProfileViewController.swift [hunk 2]
```diff
@@ -8,23 +8,57 @@
...
+        
+    // making the customized navigation controller
+    public func navController(title: String) {
+        // adding the bottom shadow
+        let shadowView = UIView(frame: self.navigationController!.navigationBar.frame)
+            shadowView.backgroundColor = UIColor.white
+            shadowView.layer.masksToBounds = false
+            shadowView.layer.shadowOpacity = 0.22
+            shadowView.layer.shadowOffset = CGSize(width: 0, height: 5)
+            shadowView.layer.shadowRadius = 3
```
Comment (carmenj) [inline Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/ProfileViewController.swift]: wrong indentation

File: Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/ProfileViewController.swift [hunk 3]
```diff
@@ -8,23 +8,57 @@
...
+        self.view.addSubview(shadowView)
+            
+        // adding the title
+        let lbNavTitle = UILabel (frame: CGRect(x: 0, y: 0, width: 320, height: 50))
+            lbNavTitle.textColor = UIColor.colorText
+            lbNavTitle.numberOfLines = 0
+            lbNavTitle.center = CGPoint(x: 0, y: 0)
+            lbNavTitle.textAlignment = .center
+            lbNavTitle.text = title
+            lbNavTitle.font = lbNavTitle.font.withSize(32)
```
Comment (carmenj) [inline Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/ProfileViewController.swift]: wrong indentation

**Review by carmenj — APPROVED** [review]

File: Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/ProfileViewController.swift [hunk 4]
```diff
@@ -6,25 +6,69 @@
...
-
-    /*
-    // MARK: - Navigation
 
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+    // making the image round and set colors to labels
+    private func configure(){
```
Comment (MunBeniamin) [inline Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/ProfileViewController.swift]: missing space after method name

File: Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/ProfileViewController.swift [hunk 5]
```diff
@@ -6,25 +6,69 @@
...
+        lbNavTitle.numberOfLines = 0
+        lbNavTitle.center = CGPoint(x: 0, y: 0)
+        lbNavTitle.textAlignment = .center
+        lbNavTitle.text = title
+        lbNavTitle.font = lbNavTitle.font.withSize(32)
+        self.navigationItem.titleView = lbNavTitle
+    }
+        
+    // sign out button
+    @IBAction func logOut(_ sender: Any) {
```
Comment (MunBeniamin) [inline Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/ProfileViewController.swift:73]: This should be done on the AuthApiManager, and on the profile page just call the logout method from the manager. The manager should notify the AppDelegate when the user is logged out and there you should call the reset root method which is already implemented.

**Review by MunBeniamin — CHANGES_REQUESTED** [review]

**Review by patriciaghitun — APPROVED** [review]

**Review by MunBeniamin — APPROVED** [review]

**Pull request merged**